src/objects/AutoNumber__c.object

```diff
@@ -0,0 +1,247 @@
+        <externalId>false</externalId>
+        <inlineHelpText>The format for the auto-numbered field including number of digits and, if desired, a prefix.</inlineHelpText>
+        <label>Display Format</label>
+        <length>30</length>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Field_API_Name__c</fullName>
+        <description>The API name of the field that contains the auto-number.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The API name of the field that contains the auto-number.</inlineHelpText>
+        <label>Field API Name</label>
+        <length>40</length>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>IsActive__c</fullName>
+        <description>When selected, the auto-number format is active.</description>
+        <defaultValue>false</defaultValue>
+        <externalId>false</externalId>
+        <inlineHelpText>When selected, the auto-number format is active.</inlineHelpText>
+        <label>Active</label>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Max_Used_Number__c</fullName>
+        <externalId>false</externalId>
+        <label>Max Used Number</label>
+        <precision>10</precision>
+        <required>false</required>
+        <scale>0</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Object_API_Name__c</fullName>
+        <description>The API name of the object for the auto-number.</description>
+        <externalId>false</externalId>
+        < inlineHelpText>The API name of the object for the auto-number.</inlineHelpText>
+        <label>Object API Name</label>
+        <length>40</length>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Starting_Number__c</fullName>
+        <description>The first number in the auto-numbered sequence.</description>
+        <defaultValue>0</defaultValue>
+        <externalId>false</externalId>
+        <inlineHelpText>The first number in the auto-numbered sequence.</inlineHelpText>
+        <label>Starting Number</label>
+        <precision>10</precision>
+        <required>false</required>
+        <scale>0</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <label>AutoNumber</label>
+    <nameField>
+        <label>Auto-Number Name</label>
+        <type>Text</type>
+    </nameField>
+    <pluralLabel>AutoNumbers</pluralLabel>
```

Comment:
@kenlewis-sfdo should this be Auto-Number Names or should it be the plural of the API name?

Reply:
This one refers to the plural label of the object itself, which is named AutoNumber__c, so I think this is ok.